scripts/pick-t2m-checkpoint.py

```
@@ -0,0 +1,94 @@
+every archived checkpoint on BOTH, restricted to the actions a human actually
+confirmed are usable, and choose deliberately.
+
+USER_GOOD below is real user feedback on v7.8 ep60, not a guess. Keep it in sync
+when new feedback arrives — and note the caveat recorded in EVAL_NOTES: the
+shoulder-line facing metric measures FACING only and does NOT capture limb
+articulation, so it ranks actions unreliably. It informs the choice between
+checkpoints of the SAME model; it does not replace looking at renders.
+"""
+import importlib.util,os,json,sys,glob,numpy as np,onnxruntime as ort,tempfile,torch
+sp=importlib.util.spec_from_file_location("rd","/Users/fernandotonon/QtMeshEditor/scripts/eval-t2m-refdist.py")
+rd=importlib.util.module_from_spec(sp); sp.loader.exec_module(rd)
```

Ruff (0.16.2)

[error] Multiple statements on one line (semicolon)

(E702)